pydatastructs/trees/tests/test_binary_trees.py

```diff
@@ -41,25 +42,25 @@ def test_BinarySearchTree():
     assert b.search(3) is None
     assert b.delete(13) is None
 
-    in_order = trav.depth_first_search(order='in_order')
+    # in_order = trav.depth_first_search(order='in_order')
     pre_order = trav.depth_first_search(order='pre_order')
-    assert [node.key for node in in_order] == [1, 4, 6, 7, 8, 14]
+    # assert [node.key for node in in_order] == [1, 4, 6, 7, 8, 14]
     assert [node.key for node in pre_order] == [8, 4, 1, 6, 7, 14]
 
     b.delete(7)
     b.delete(6)
     b.delete(1)
     b.delete(4)
 
-    in_order = trav.depth_first_search(order='in_order')
+    # in_order = trav.depth_first_search(order='in_order')
```

Comment:
Why `in_order` tests are commented out?

Reply:
Yet to be done.

Reply:
I'm doing this, will be done within 1/2 an hour

Reply:
inorder traversal implemented and tested! Working fine